{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"init"},{"kind":"text","text":"("},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? = nil)"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsapplicationdiscoveryservice\/serverinternalerrorexception\/init(message:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService\/ServerInternalErrorException\/init(message:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"("},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?)"}],"title":"init(message:)","roleHeading":"Initializer","role":"symbol","symbolKind":"init","externalID":"s:30AWSApplicationDiscoveryService28ServerInternalErrorExceptionV7messageACSSSg_tcfc","modules":[{"name":"AWSApplicationDiscoveryService"}]},"hierarchy":{"paths":[["doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService","doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService\/ServerInternalErrorException"]]},"references":{"doc://AWSApplicationDiscoveryService/documentation/AWSApplicationDiscoveryService/ServerInternalErrorException":{"role":"symbol","title":"ServerInternalErrorException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ServerInternalErrorException"}],"abstract":[{"type":"text","text":"The server experienced an internal error. Try again."}],"identifier":"doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService\/ServerInternalErrorException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ServerInternalErrorException"}],"url":"\/documentation\/awsapplicationdiscoveryservice\/serverinternalerrorexception"},"doc://AWSApplicationDiscoveryService/documentation/AWSApplicationDiscoveryService":{"role":"collection","title":"AWSApplicationDiscoveryService","abstract":[],"identifier":"doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService","kind":"symbol","type":"topic","url":"\/documentation\/awsapplicationdiscoveryservice"},"doc://AWSApplicationDiscoveryService/documentation/AWSApplicationDiscoveryService/ServerInternalErrorException/init(message:)":{"role":"symbol","title":"init(message:)","fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"("},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?)"}],"abstract":[],"identifier":"doc:\/\/AWSApplicationDiscoveryService\/documentation\/AWSApplicationDiscoveryService\/ServerInternalErrorException\/init(message:)","kind":"symbol","type":"topic","url":"\/documentation\/awsapplicationdiscoveryservice\/serverinternalerrorexception\/init(message:)"}}}